### DISCUSSION TITLE
Q A

### DISCUSSION THREAD

**Discussion by enzifiri** [enzifiri/GitHub-Achievements#87] [Q&A] [answered]
Please read carefully our "Zinnia Crew3 Quest Guide" to find out how much you will be able to earn.

**Comment by zagiscoming** [comment, marked as answer]
thx for all

**Comment by Vevivo** [comment]
emeğine sağlık hocam

**Comment by NuriBartu05** [comment]
hocam teşekkürler yazı için

**Comment by Secure-Sentinel** [comment]
Emeklerine sağlık brocum
